bigtable/CMakeLists.txt

```diff
@@ -122,8 +125,79 @@ target_link_libraries(googleapis ${GRPCPP_LIBRARIES} ${GRPC_LIBRARIES} ${PROTOBU
 # Enable unit tests
 enable_testing()
 
+# This is an extremely ugly hack to link with Abseil code
```

Comment:
Indeed, I do not want to maintain a copy of the BUILD file / Makefile for cctz and abseil, they should distribute their own.

Reply:
reverted.